im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,426 @@
+      <term name="presented at">präsentiert bei</term>
+      <term name="retrieved">abgerufen am</term>
+      <term name="available at">verfügbar bei</term>
+      <term name="scale">Skala</term>
+      <term name="open-quote">»</term>
+      <term name="close-quote">«</term>
+      <term name="open-inner-quote">›</term>
+      <term name="close-inner-quote">‹</term>
+    </terms>
+  </locale>
+  <macro name="editor">
+    <names variable="editor" delimiter=": ">
+      <label form="short" text-case="sentence" suffix=" von "/>
+      <name delimiter="/" suffix=". " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="anon">
+    <text term="anonymous" form="short" strip-periods="false"/>
+  </macro>
+  <macro name="author">
+    <names variable="author" suffix=": ">
+      <name delimiter="/" delimiter-precedes-last="always" initialize="false" sort-separator=""/>
+      <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
+      <substitute>
+        <names variable="editor" suffix=": "/>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" delimiter="/" delimiter-precedes-last="always" et-al-min="4" et-al-use-first="2" initialize-with=". "/>
+      <substitute>
+        <names variable="editor">
+          <name form="short" delimiter="/" delimiter-precedes-last="always" et-al-min="4" et-al-subsequent-min="2"/>
+        </names>
+        <names variable="collection-editor">
+          <name form="short" et-al-min="4" et-al-subsequent-min="2"/>
+        </names>
+        <text macro="anon"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="URL">
+        <group>
+          <text variable="URL"/>
+          <group prefix=" [" suffix="]">
+            <date variable="accessed">
+              <date-part name="day" form="numeric-leading-zeros" suffix="."/>
+              <date-part name="month" form="numeric-leading-zeros" suffix="."/>
+              <date-part name="year"/>
+            </date>
+          </group>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="text" variable="issued" prefix=" ">
+          <date-part name="year"/>
+        </date>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="pages">
+    <choose>
+      <if type="article-journal article-magazine article-newspaper broadcast thesis webpage chapter paper-conference" match="any">
+        <label strip-periods="true" prefix=", " suffix=". " variable="page" form="short"/>
+        <text variable="page"/>
+        <text variable="note" prefix=" [erstmals " suffix="]"/>
```

Comment:
I'm using the note for the original publication date. As far as I know there is no other option to include it, is there?